test-data/unit/check-typevar-defaults.test

```diff
@@ -239,3 +239,164 @@ def func_c4(
     # reveal_type(b)  # Revealed type is "__main__.ClassC4[builtins.int, builtins.str]"  # TODO
     reveal_type(c)  # N: Revealed type is "__main__.ClassC4[builtins.int, builtins.float]"
 [builtins fixtures/tuple.pyi]
+
+[case testTypeVarDefaultsTypeAlias1]
+# flags: --disallow-any-generics
+from typing import Any, Dict, List, Tuple, TypeVar, Union
+
+T1 = TypeVar("T1")
+T2 = TypeVar("T2", default=int)
+T3 = TypeVar("T3", default=str)
+T4 = TypeVar("T4")
+
+TA1 = Dict[T2, T3]
+
+def func_a1(
+    a: TA1,  # E: Missing type parameters for generic type "TA1"
```

Comment:
This errors seems like a false positive, since both of the type parameters have a default.

Reply:
Good catch! I added the `--disallow-any-generics` flag specifically to detect it, but somehow missed to check it. Pushed a new commit.